linux_os/guide/system/network/network-firewalld/firewalld_activation/package_firewalld_installed/rule.yml

```diff
@@ -15,11 +15,15 @@ identifiers:
     cce@rhel7: CCE-82999-4
     cce@rhel8: CCE-82998-6
     cce@rhcos4: CCE-82521-6
+    cce@sle15: CCE-85698-9
 
 references:
     nist: CM-6(a)
+    nist@sle15: CM-7,CM-7.1(iii),CM-7(b),AC-17(1)
     srg: SRG-OS-000480-GPOS-00227,SRG-OS-000298-GPOS-00116
+    srg@sle15: SRG-OS-000096-GPOS-00050,SRG-OS-000297-GPOS-00115,SRG-OS-000480-GPOS-00232
     cis@rhel8: 3.4.1.1
+    stigid@sle15:  SLES-15-010220
```

Comment:
The same STIG id is assigned to `service_firewalld_enabled`. That's not ideal and we already requested DISA to split these for RHEL8. That's probably not going to be reflected to the SLE STIGs. Having a duplicate means your STIG results will map to only one of these if you import results into STIG Viewer for example. On the other hand, for openscap html reports, there will be no negative impact.

Reply:
I'm not opposed to merging this, I just want to have your acknowledgement.

Reply:
acknowledged and agreed. Not quite sure what else to do given our stigs. it is better than what we have in SLE-12 where we have three rules attached to the same stig.

Reply:
@ggbecker let's go forward as it is. I was told that getting changes in the STIGs takes a while. We only have two STIGs related to firewalld in SLE-15 SLES-15-010220 and SLE-15 SLES-15-010370.
SLE-15 SLES-15-010220 really covers configuration which we attached to install as well, since we can't configure what is not installed.

SLE-15 SLES-15-010370 covers testing the panic feature to disable remote access. Not sure how to test that one. Direction really requires being attached to system console.